packages/vertix-bot/.env-local

```diff
@@ -4,7 +4,7 @@ BOT_PRISMA_DATABASE_URL="mongodb://127.0.0.1:27017/discord?directConnection=true
 
 # Debug of cache
 #DEBUG_CACHE="
-Vertix/Managers/TopGG
+VertixBot/Managers/TopGG
```

Comment:
/ask create a todo issue to check all .env usages and update `.env-local` according to the changes

Reply:
Handled by #81